modules/planning/common/reference_line_info.cc

```diff
@@ -147,13 +147,25 @@ bool ReferenceLineInfo::GetFirstOverlap(
 
 void ReferenceLineInfo::InitFirstOverlaps() {
   const auto& map_path = reference_line_.map_path();
+  // clear_zone
+  hdmap::PathOverlap clear_area_overlap;
+  if (GetFirstOverlap(map_path.clear_area_overlaps(), &clear_area_overlap)) {
+    first_encounter_overlaps_.push_back({CLEAR_AREA, clear_area_overlap});
+  }
 
   // crosswalk
   hdmap::PathOverlap crosswalk_overlap;
   if (GetFirstOverlap(map_path.crosswalk_overlaps(), &crosswalk_overlap)) {
     first_encounter_overlaps_.push_back({CROSSWALK, crosswalk_overlap});
   }
 
+  // pnc_junction
+  hdmap::PathOverlap pnc_junction_overlap;
+  if (GetFirstOverlap(map_path.pnc_junction_overlaps(),
+                      &pnc_junction_overlap)) {
+    first_encounter_overlaps_.push_back({PNC_JUNCTION, pnc_junction_overlap});
+  }
```

Comment:
emplace_back(PNC_JUNCTION, pnc_junction_overlap) will be more efficient.

Reply:
good point. thx! amended